core/src/main/java/org/apache/druid/math/expr/Function.java

```diff
@@ -25,6 +25,7 @@
 import org.apache.druid.java.util.common.IAE;
 import org.apache.druid.java.util.common.RE;
 import org.apache.druid.java.util.common.StringUtils;
+import org.apache.druid.math.expr.ExprEval.ArrayExprEval;
```

Comment:
This line might get flagged as an unused import.

Reply:
Oops, fixed.